apps/examples/wifi_manager/wm_test/easysetup_test_guide.md

```diff
@@ -0,0 +1,24 @@
+Setup guide:-
+a)Build TP1x source code in with rtl8721csm/security_hal_test_tz config
+b)H/W Requirement:
+	1)Access Point
+	2)Device under test (DUT) – TP1x board - Simulating easy setup process on Samsung DA device
+	3)Client Device - TP1x board - Simulating easy setup process on mobile SmartThings Application 
+c)Add hostname in input file
+	wm_test add_hostname [filepath] [hostname]
+	e.g. wm_test add_hostname /mnt/wt_hostname.txt www.samsung.com 
+	
+Test guide:
+DUT(Easysetup Server): 
+wm_test easysetup_server [ap ssid] [ap security mode] [ap password] [softap ssid] [softap password] [softap channel] [repeat count] [udp/dtls]* [udp/dtls data size in KB]** [tcp/tls]* [tcp/tls data size in KB]** [hostname filepath]***
+e.g. wm_test easysetup Infa_6A_3 wpa2_aes 12345678 tp1x 12341234 1 3 udp 100 tcp 100 /mnt/wt_hostname.txt
+
+Client device: 
+wm_test client [ap ssid] [ap security mode] [ap password] [softap ssid] [softap password] [softap channel] [repeat count] [udp/dtls]* [udp/dtls data size in KB]** [tcp/tls]* [tcp/tls data size in KB]**
+e.g. wm_test client Infa_6A_3 wpa2_aes 12345678 tp1x 12341234 1 3 udp 100 tcp 100
+
```

Comment:
How about to add overall test list in the document.
for example this test covers following items
UDP, TCP, DTLS, TLS, Wi-Fi connect disconnect

Reply:
test command initializes wi-fi manager internally.
there is not separate command for Wi-Fi connect & disconnect

Reply:
Changes Done as suggested.